src/e2e/java/teammates/e2e/pageobjects/GoogleLoginPage.java

```diff
@@ -0,0 +1,132 @@
+package teammates.e2e.pageobjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import com.google.common.base.Preconditions;
+
+import teammates.e2e.util.TestProperties;
+
+/**
+ * Page Object Model for login page using official Google account.
```

Comment:
It's not a "login page using official Google account". It's the "official Google Accounts login page".